src/components/app/data/services/contentHighlights.js

```diff
@@ -2,6 +2,7 @@ import { camelCaseObject, getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 import { logError } from '@edx/frontend-platform/logging';
 
+import MAX_HIGHLIGHT_SETS from '../constants';
```

Comment:
```suggestion
import { MAX_HIGHLIGHT_SETS } from '../constants';
```
For fixing error
![image](https://github.com/user-attachments/assets/b9390618-be37-429f-9653-08471c819329)